base/reflection.jl

```diff
@@ -542,14 +546,18 @@ code_llvm(f::ANY, types::ANY=Tuple) = code_llvm(STDOUT, f, types)
 code_llvm_raw(f::ANY, types::ANY=Tuple) = code_llvm(STDOUT, f, types, false)
 
 """
-    code_native([io], f, types)
+    code_native([io], f, types, [asm_variant])
```

Comment:
d'oh, didn't check which branch I was on before looking at docs locally! so @citisolo this change to the signature would need to be made in the rst copy